assets/js/src/core/components/editor-tabs/editor-tabs.tsx

```diff
@@ -75,6 +80,24 @@ export const EditorTabs = ({ defaultActiveKey, showLabelIfActive, items }: IEdit
     detachWidget(item)
   }
 
+  items = items.filter((item) => {
+    if (item.hidden !== undefined && !item.hidden()) {
```

Comment:
shouldn't this be the otherway round? so `item.hidden()` instead of `!item.hidden()`?

Reply:
true - corrected it :+1: